User/user.md

```diff
@@ -0,0 +1,157 @@
+# User
+
+## Overview
+
+ A User in OpenMRS is an account that a person may use to log into the system.
+
+The real-life person is represented by a Person record in OpenMRS, and a person may have more than one user account. If you want a patient to be able to view her own record in OpenMRS, then you need to create a user account and link it to the patient.
+
+## Available operations
+
+1. [List users](#list-user)
+2. [Create a user](#create-a-user)
+3. [Update a user](#update-a-user)
+4. [Delete a user](#delete-a-user)
+
+
+### List user
+
+* #### List all non-voided users.
```

Comment:
Users are [retired](https://docs.openmrs.org/datamodel/openmrs-refapp-data-model-1.9.html#users.retire), not voided.